tests/roxctl/bats-tests/local/expect/flavor-interactive-postgres.expect.tcl

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/expect -f
```

Comment:
How are changes in this file related to the declarative configuration?

Reply:
So, long story short, with the introduction of the two new flags on the `generate` command, we will have more output in `interactive` mode.

Now, I realized when doing the expect for both postgres and non-postgres, that there are more questions for postgres. Instead of adding yet another if's within the expect file, I simply resorted to creating a separate expect for "postgres" mode, which also asserts for the declarative configuration questions.